Add Gitlab set up instructions for Code Coverage product (#31260)
DataDog/documentation: content/en/code_coverage/data_collected.md

[1]: https://docs.github.com/en/webhooks/webhook-events-and-payloads

{{% /tab %}}
{{% tab "Gitlab" %}}

> Code Coverage relies on the Gitlab webhooks. None of the webhooks include your source code content; they only include metadata about the merge request, such as title, description, author, labels, and commit SHAs.

Reviewer: ```suggestion
Code Coverage relies on Gitlab webhooks. The webhooks do not include your source code content. They only include metadata about the merge request, such as the title, description, author, labels, and commit SHAs.
```

> See Gitlab's [webhook events and payloads documentation][1] for a detailed description of the data sent by the webhooks.

Reviewer: ```suggestion
See Gitlab's [webhook events and payloads documentation][1] for a detailed description of the data sent by webhooks.
```
